docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -759,6 +759,20 @@ The following command operators are supported:
     JSON struct or list is rendered. Structs and lists may be nested. In any event, the maximum
     length is ignored
 
+%UPSTREAM_FILTER_STATE(KEY:F):Z%
+  HTTP
+    :repo:`StreamInfo <envoy/stream_info/stream_info.h>` could contain filter state for upstream
```

Comment:
I think the repo link may not be appropriate for this audience. I think you should mostly copy the docs for `%FILTER_STATE(KEY:F):Z%` from this file, with minor changes to note it's for upstream.